rhombus/scribblings/ref-string.scrbl

```diff
@@ -120,6 +121,21 @@ immutable strings.
 
 }
 
+@doc(
+  fun String.contains(str :: ReadableString, substr :: ReadableString) :: Boolean
+){
+
+ Returns whether or not @rhombus(str) contains the string @rhombus(substr)
```

Comment:
A stylistic issue: predicates are conventionally documented (as in Racket) as “Returns `#true` if \<some condition\>, `#false` otherwise.”  If you think the phrasing “whether” is better, consider changing the verb to “Checks”.  Also note the missing full stop.